### PR TITLE
Read me update

### PR DESCRIPTION
**Pull request by wesboyt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +35,40 @@ For more information on Ion have a look at the [Ion Specification](https://amzn.
...
+
+| Types        | IonText | IonBinary | Limitations  |
+|:-------------:|:-------------:|:-------------:|:-------------:|
+| null      | yes | no      | none |
+| bool      | yes      | no      |   none |
+| int | yes      | no      |    underscores, binaryints, bigints |
+| float | yes      | no      |    underscores |
+| decimal | yes      | no      |    none |
+| timestamp | yes      | no      |    offsets/max values non spec-compliant |
+| string | yes      | no      |    values outside of ucs2, escapes between triplequotes |
```
Comment (tgregg) [inline README.md]: What's an example of an "escape between triple quotes"?

File: README.md [hunk 2]
```diff
@@ -9,6 +9,10 @@
  <a title="semantic-release" href="https://github.com/amzn/ion-js/releases"><img src="https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg?style=flat-square"/></a>
 </p>
 
+
+
+
+
```
Comment (tgregg) [inline README.md]: Why all the blank lines?

File: README.md (hunk 1 above)
Comment (wesboyt) [reply README.md]: should be on the github issue

File: README.md (hunk 2 above)
Comment (wesboyt) [reply README.md]: removed

**Pull request merged**